test/Chainweb/Test/Pact/PactMultiChainTest.hs

```diff
@@ -1044,6 +1045,28 @@ chainweb219UpgradeTest = do
     tryReadMsg = buildBasicGas 1000
         $ mkExec' "(try 1 (read-msg \"somekey\"))"
 
+pact48UpgradeTest :: PactTestM ()
+pact48UpgradeTest = do
+  runToHeight 83 -- TODO: What height do we want pre-fork, post-fork?
```

Comment:
This height just needs to be a few blocks past the last fork.

Reply:
Thanks! Forgot to remove my TODO. fixed now.